src/rom_workflows.cc

```diff
@@ -512,7 +512,7 @@ void testROMRhoOperator(MGmolInterface *mgmol_)
     MGmol<OrbitalsType> *mgmol = static_cast<MGmol<OrbitalsType> *>(mgmol_);
     Poisson *poisson = mgmol->electrostat_->getPoissonSolver(); 
     Potentials& pot = mgmol->getHamiltonian()->potential();
-    std::shared_ptr<Rho<OrbitalsType>> rho = mgmol->getRho();
+    std::shared_ptr<Rho<OrbitalsType>> rho = NULL; // mgmol->getRho();
```

Comment:
@dreamer2368 I got a build error from this line. Is this function still under development or did I miss something?